var/spack/repos/builtin/packages/libxc/package.py

```diff
@@ -63,31 +67,38 @@ def libs(self):
             libraries, root=self.prefix, shared=shared, recursive=True
         )
 
-    def install(self, spec, prefix):
+    def setup_environment(self, spack_env, run_env):
         # Optimizations for the Intel compiler, suggested by CP2K
         optflags = '-O2'
         if self.compiler.name == 'intel':
-            optflags += ' -xAVX -axCORE-AVX2 -ipo'
+            # Oh flake8 E265, why won't you let us distinguish commented-out
+            # code from discursive comments?
+            #
+            # optflags += ' -xAVX -axCORE-AVX2 -ipo'
+            #
+            # Well, not every lowly login node has AVX or AVX2:
+            #      $ icc  -xAVX -axCORE-AVX2 -ipo hello.c
+            #      $ ./a.out
+            #      Please verify that both the operating system and the \
+            #      processor support Intel(R) AVX instructions.
+            #
+            # TODO: This really should be handled via variants.
```

Comment:
in `dealii` we handle this as an additional variant `optflags`.

Reply:
Sorry, they syntax would be `icc  -xSSE4.2 -axAVX,CORE-AVX2`. You can specify as many alternate paths as you like, but with a single, comma-separated -ax arg.